src/scss/App.scss

```diff
@@ -52,6 +53,20 @@ limitations under the License.
 @import '@tektoncd/dashboard-components/dist/components/Trigger/Trigger.scss';
 @import '@tektoncd/dashboard-components/dist/components/ViewYAML/ViewYAML.scss';
 
+html, #root, .tkn--main-content {
+  height: 100%;
+}
+
+body {
+  // to prevent unwanted scroll when content doesn't fill full height
+  height: calc(100% - 3rem);
+}
+
+#main-content {
+  //height of the main header, added for accessibility of skip to main content link
```

Comment:
super-nit: add a space between slashes and height

Reply:
updated, thanks